modules/memegenBidAdapter.js

```diff
@@ -47,28 +50,61 @@ var MemeGenAdapter = function MemeGenAdapter() {
     }
 
     // build bid request with impressions
-    var bidRequest = {
-      id: utils.getUniqueIdentifierStr(),
-      imp: [{
-        id: bidReq.bidId,
-        banner: {
-          w: adW,
-          h: adH
+    var bidRequest;
+    if (isInapp) {
+      bidRequest = {
```

Comment:
can you extract a mehtod of `buildInappRequest` and `buildWebRequst`?

Reply:
I thought of it, but the passing the values will be much more annoing than just an if (which is very simple and can be even replaced by ternary operator)

Reply:
I prefer to extract a method with global params